src/test/java/fi/metatavu/metaform/server/test/functional/tests/AdminThemeTestsIT.kt

```diff
@@ -0,0 +1,127 @@
+package fi.metatavu.metaform.server.test.functional.tests
+
+import fi.metatavu.metaform.server.persistence.model.Metaform
+import fi.metatavu.metaform.server.test.functional.AbstractTest
+import fi.metatavu.metaform.server.test.functional.builder.TestBuilder
+import fi.metatavu.metaform.server.test.functional.builder.resources.KeycloakResource
+import fi.metatavu.metaform.server.test.functional.builder.resources.MysqlResource
+import fi.metatavu.metaform.server.test.functional.tests.GeneralTestProfile
+import io.quarkus.test.common.QuarkusTestResource
+import io.quarkus.test.junit.QuarkusTest
+import io.quarkus.test.junit.TestProfile
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+/**
+ * Tests for Admin Theme API
+ * @author Otto Hooper
+ */
+@QuarkusTest
+@QuarkusTestResource.List(
+        QuarkusTestResource(MysqlResource::class),
+        QuarkusTestResource(KeycloakResource::class)
+)
+@TestProfile(GeneralTestProfile::class)
+class AdminThemeTestsIT : AbstractTest() {
+    @Test
+    @Throws(Exception::class)
+    fun createAdminThemeTest() {
+        TestBuilder().use { builder ->
+            val adminTheme = builder.metaformAdmin.adminThemes.create(
+                    data = "data",
+                    name = "Test admin theme",
+                    slug = "test-admin-theme"
+            )
+            
+            Assertions.assertNotNull(adminTheme)
+            Assertions.assertNotNull(adminTheme.id)
+            Assertions.assertNotNull(adminTheme.data)
+            Assertions.assertNotNull(adminTheme.name)
+            Assertions.assertNotNull(adminTheme.slug)
+        }
```

Comment:
Find the created theme and assert equal with adminTheme returned from the create test for all fields

Reply:
With that, you dont need a separate find test